cibuild.cmd

```diff
@@ -125,6 +126,7 @@ echo ** Moving bootstrapped MSBuild to the bootstrap folder
 taskkill /F /IM vbcscompiler.exe
 
 set MSBUILDLOGPATH=%~dp0msbuild_move_bootstrap-%HOST%.log
+set MSBUILDBINARYLOGPATH=%~dp0msbuild_move_bootstrap-%HOST%.binlog
```

Comment:
The file logger seems unnecessary now right?. Can you please remove it?

Reply:
@cdmihai You're saying we should no longer create a text log?

Reply:
Initially I thought that having the binary log makes the text one obsolete. But now that I think of it, having two loggers piped through the msbuild app makes for a nice test case. So let's leave both of them :)